corehq/apps/translations/app_translations.py

```diff
@@ -785,6 +794,7 @@ def has_translation(row_, langs):
                   " for the label '%s' in sheet '%s'") % (label, sheet.worksheet.title)
             ))
     # Update the translations
+    missing_cols = _get_missing_cols(app, sheet)
```

Comment:
nit: comment should probably be under this line

Reply:
(resolved by later commit)